### PR TITLE
Update installation.rst

### PR DESCRIPTION
**Pull request by ua0088**
06.08.2024 - changes added for Installation section, including for Redhat

### CODE REVIEW

File: admin/installation.rst [hunk 1]
```diff
@@ -376,9 +376,9 @@ Provided driver packages:
 
   * netxms-dbdrv-pgsql - PostgreSQL driver
   * netxms-dbdrv-mariadb - Mariadb driver
-  * netxms-dbdrv-mysql - MySQL driver (not built for Ubuntu 20 / Mint 20)
+  * netxms-dbdrv-mysql - MySQL driver, currently under development (not built for Ubuntu 20 / Mint 20)
   * netxms-dbdrv-odbc - unixODBC driver (can be used with DB/2 and Microsoft SQL)
-  * netxms-dbdrv-oracle - Oracle driver
+  * netxms-dbdrv-oracle - Oracle driver, currently under development ( requires Oracle client installation )
```
Comment (alkk) [inline admin/installation.rst]: it's currently built for all available platforms -- amd64 version of the oracle client is still unavailable

File: admin/installation.rst [hunk 2]
```diff
@@ -241,11 +241,11 @@ However, we test and officially support only some of them.
 Supported platforms for |product_name| server and agent:
 
    * Debian 10 (Buster), 11 (Bullseye), 12 (Bookworm)
-   * Ubuntu 18.04 LTS (Bionic), 20.04 LTS (Focal Fossa), 22.04 LTS (Jammy Jellyfish)
+   * Ubuntu 18.04 LTS (Bionic), 20.04 LTS (Focal Fossa), 22.04 LTS (Jammy Jellyfish), 24.04 ( Noble )
```
Comment (filippsudanov) [inline admin/installation.rst]: Let's make it like this for consistency:

Ubuntu 18.04 LTS (Bionic), 20.04 LTS (Focal), 22.04 LTS (Jammy), 24.04 (Noble)

File: admin/installation.rst [hunk 3]
```diff
@@ -282,7 +282,7 @@ Database
 
 Database engines supported by |product_name| server:
 
-   * PostgreSQL 9.5, 9.6, 10, 11, 12, 13, 14
+   * PostgreSQL 9.5, 9.6, 10, 11, 12, 13, 14, 16
```
Comment (filippsudanov) [inline admin/installation.rst]: Probably 15 as well

File: admin/installation.rst [hunk 4]
```diff
@@ -484,13 +484,13 @@ Web Management Client
 ^^^^^^^^^^^^^^^^^^^^^
 
 |product_name| web interface is java based and should be deployed into servlet container to
-run. Minimal supported versions: Jetty 10, Tomcat 9. Supported Java version is 17. 
+run. Minimal supported versions: Jetty 10, Tomcat 9. Supported Java version is 17, but is found to be working with later versions, for example 21.
```
Comment (filippsudanov) [inline admin/installation.rst]: I'd just say:

Supported Java version is 17 or later

File: admin/installation.rst [hunk 5]
```diff
@@ -532,6 +532,148 @@ DNF provide simple way to add repository:
...
+
+
+Agent
+~~~~~
+
+Install core agent package ("netxms-agent") and optional subagent packages, if required:
+
+.. code-block:: sh
+
+  apt-get install netxms-agent
```
Comment (filippsudanov) [inline admin/installation.rst]: should be dnf

File: admin/installation.rst [hunk 6]
```diff
@@ -466,7 +466,7 @@ Due to limitation of Eclipse platform used to build the Management Client, only
  
  2. Download the latest version from http://www.netxms.org/download. You will need
```
Comment (filippsudanov) [inline admin/installation.rst]: This should be updated to current situation:
- download .jar file
- run it using java -jar nxmc-xxx.jar

File: admin/installation.rst [hunk 7]
```diff
@@ -532,6 +532,148 @@ DNF provide simple way to add repository:
...
+~~~~~~~~~~~~~~~~~
+
+Desktop Management Client
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Due to limitation of Eclipse platform used to build the Management Client, only x64 build is provided.
+
+ 1. Make sure you have 64-bit Java version 17 installed you your system. 
+ 
+ 2. Download the latest version from http://www.netxms.org/download. You will need
```
Comment (filippsudanov) [inline admin/installation.rst]: This should be updated to current situation:
- download .jar file
- run it using java -jar nxmc-xxx.jar

**Pull request merged**